.rubocop.yml

```diff
@@ -1,13 +1,51 @@
 inherit_from: .rubocop_todo.yml
 
-# rspec expect{...} is conventional
-Style/SpaceBeforeBlockBraces:
+AllCops:
+  DisplayCopNames: true
+  Include:
+    - Rakefile
+    - lib/**/*.rake
+  Exclude:
+    - Gemfile.lock
+    - '**/*.md'
+    # - bin/**/*
+    - config/**/*.yml
+    # - config/environments/*.rb
+    - script/**/*
+    - vendor/**/*
+
+Bundler/OrderedGems:
+  Exclude:
+    - Gemfile
```

Comment:
This is the only file that `Bundler/OrderedGems` applies to, so perhaps turning it off entirely would be more clear.

Reply:
Good point - filed away for future reference.